keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -882,81 +818,60 @@ See Drawbacks section
-but the size is reduced considerable.
+but the size of the objects stored is reduced considerable.
 
 ###### Will enabling / using this feature result in introducing new API types?
 
 See [API](#api)
 
 ###### Will enabling / using this feature result in any new calls to the cloud provider?
 
-N/A
+No
 
 ###### Will enabling / using this feature result in increasing size or count of the existing API objects?
 
-See [Drawbacks](#drawbacks)
+See [Drawbacks](#drawbacks), a new IPAddress object will be created by Service.Spec.ClusterIP
 
 ###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
 
-<!--
-Look at the [existing SLIs/SLOs].
-
-Think about adding additional work or introducing new steps in between
-(e.g. need to do X to start a container), etc. Please describe the details.
-
-[existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
--->
+No
 
 ###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
-The apiservers will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
+The apiservers and cloud-controller manager will increase the memory usage because they have to keep a local informer with the new objects ServiceCIDR and IPAddress.
 
 ### Troubleshooting
 
-<!--
-This section must be completed when targeting beta to a release.
-
-For GA, this section is required: approvers should be able to confirm the
-previous answers based on experience in the field.
-
-The Troubleshooting section currently serves the `Playbook` role. We may consider
-splitting it into a dedicated `Playbook` document (potentially with some monitoring
-details). For now, we leave it here.
--->
-
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
-The feature depends on the API server to work.
+The feature depends on the API server to work, however, the components consuming the existing objects will be unaffected.
 
 ###### What are other known failure modes?
 
-<!--
-For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
--->
+- Can not create Service with ClusterIP
+  - Detection: via metrics or because an Create Service API call fails
```

Comment:
logs trace it takes time to allocate an IP
```suggestion
  - Detection: via metrics or because an Create Service API call fails
pkg/registry/core/service/ipallocator/ipallocator.go
> 	trace := utiltrace.New("allocate dynamic ClusterIP address")
>	defer trace.LogIfLong(500 * time.Millisecond)
```